// @carbon/pictograms-react/es/printer/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _objectWithoutProperties, I as Icon, a as _extends } from '../Icon-ca73eb7b.js';
import React from 'react';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;

var _excluded = ["children"];
var Printer = /*#__PURE__*/React.forwardRef(function Printer(_ref, ref) {
  var children = _ref.children,
      rest = _objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M27,31.36H5c-0.199,0-0.36-0.161-0.36-0.36v-6.64H1c-0.199,0-0.36-0.161-0.36-0.36V10\tc0-0.199,0.161-0.36,0.36-0.36h3.64V1c0-0.199,0.161-0.36,0.36-0.36h22c0.199,0,0.36,0.161,0.36,0.36v8.64H31\tc0.199,0,0.36,0.161,0.36,0.36v14c0,0.199-0.161,0.36-0.36,0.36h-3.64V31C27.36,31.199,27.199,31.36,27,31.36z M5.36,30.64h21.28\tV18.36H5.36V30.64z M27.36,23.64h3.279V10.36H1.36v13.28h3.28V18c0-0.199,0.161-0.36,0.36-0.36h22c0.199,0,0.36,0.161,0.36,0.36\tV23.64z M5.36,9.64h21.28V1.36H5.36V9.64z M27,14.36h-4v-0.72h4V14.36z"
  })), children);
});

export { Printer as default };
