// @carbon/pictograms-react/lib/tornado/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var Icon = require('../Icon-9b18a60b.js');
var React = require('react');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;

var _excluded = ["children"];
var Tornado = /*#__PURE__*/React__default["default"].forwardRef(function Tornado(_ref, ref) {
  var children = _ref.children,
      rest = Icon._objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React__default["default"].createElement(Icon.Icon, Icon._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M17.5,30.36h-4v-0.72h4V30.36z M23.5,26.36h-8v-0.72h8V26.36z M25.5,22.36h-12v-0.72h12V22.36z M21.5,18.36\th-12v-0.72h12V18.36z M28.5,14.36h-17v-0.72h17V14.36z M25.5,10.36h-17V9.64h17V10.36z M23.5,6.36h-20V5.64h20V6.36z M26.5,2.36h-20\tV1.64h20V2.36z"
  })), children);
});

module.exports = Tornado;
