// @carbon/pictograms-react/lib/cloud-pak--for-applications/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var Icon = require('../Icon-9b18a60b.js');
var React = require('react');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;

var _excluded = ["children"];
var CloudPakForApplications = /*#__PURE__*/React__default["default"].forwardRef(function CloudPakForApplications(_ref, ref) {
  var children = _ref.children,
      rest = Icon._objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React__default["default"].createElement(Icon.Icon, Icon._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36c-0.059,0-0.117-0.015-0.171-0.043l-13-7C2.713,24.254,2.64,24.133,2.64,24V8\tc0-0.132,0.073-0.254,0.189-0.317l13-7c0.107-0.058,0.234-0.058,0.342,0l13,7C29.287,7.746,29.36,7.868,29.36,8v16\tc0,0.133-0.073,0.254-0.189,0.317l-13,7C16.117,31.346,16.059,31.36,16,31.36z M3.36,23.785L16,30.591l12.64-6.806V8.215L16,1.409\tL3.36,8.215V23.785z M23,22.36H13c-0.199,0-0.36-0.161-0.36-0.36v-2.64H9c-0.199,0-0.36-0.161-0.36-0.36v-9\tc0-0.199,0.161-0.36,0.36-0.36h10c0.199,0,0.36,0.161,0.36,0.36v2.64H23c0.199,0,0.36,0.161,0.36,0.36v9\tC23.36,22.199,23.199,22.36,23,22.36z M13.36,21.64h9.28v-8.28h-9.28V21.64z M9.36,18.64h3.28V13c0-0.199,0.161-0.36,0.36-0.36h5.64\tv-2.28H9.36V18.64z"
  })), children);
});

module.exports = CloudPakForApplications;
