// @carbon/pictograms-react/lib/fog/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var Icon = require('../Icon-9b18a60b.js');
var React = require('react');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;

var _excluded = ["children"];
var Fog = /*#__PURE__*/React__default["default"].forwardRef(function Fog(_ref, ref) {
  var children = _ref.children,
      rest = Icon._objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React__default["default"].createElement(Icon.Icon, Icon._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,26.36h-4v-0.72h4V26.36z M22,26.36h-4v-0.72h4V26.36z M14,26.36h-4v-0.72h4V26.36z M6,26.36H2v-0.72h4\tV26.36z M26,22.36h-4v-0.72h4V22.36z M18,22.36h-4v-0.72h4V22.36z M10,22.36H6v-0.72h4V22.36z M30,18.36h-4v-0.72h4V18.36z M22,18.36h-4v-0.72h4V18.36z M14,18.36h-4v-0.72h4V18.36z M6,18.36H2v-0.72h4V18.36z M26,14.36h-4v-0.72h4V14.36z M18,14.36h-4\tv-0.72h4V14.36z M10,14.36H6v-0.72h4V14.36z M30,10.36h-4V9.64h4V10.36z M22,10.36h-4V9.64h4V10.36z M14,10.36h-4V9.64h4V10.36z M6,10.36H2V9.64h4V10.36z M26,6.36h-4V5.64h4V6.36z M18,6.36h-4V5.64h4V6.36z M10,6.36H6V5.64h4V6.36z"
  })), children);
});

module.exports = Fog;
