// @carbon/pictograms-react/lib/firewall/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var Icon = require('../Icon-9b18a60b.js');
var React = require('react');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;

var _excluded = ["children"];
var Firewall = /*#__PURE__*/React__default["default"].forwardRef(function Firewall(_ref, ref) {
  var children = _ref.children,
      rest = Icon._objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React__default["default"].createElement(Icon.Icon, Icon._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M28.427,20.917l-0.51,0.51l2.214,2.213H23.36V8.36H31V7.64h-7.64V1c0-0.199-0.161-0.36-0.36-0.36H11\tc-0.199,0-0.36,0.161-0.36,0.36v30c0,0.199,0.161,0.36,0.36,0.36h12c0.199,0,0.36-0.161,0.36-0.36v-6.64h6.771l-2.214,2.213\tl0.51,0.51L31.509,24L28.427,20.917z M22.64,30.64H11.36V1.36h11.28v6.28H17v0.72h5.64v15.28H17v0.721h5.64V30.64z M1,23.64h7.616\tv0.721H1V23.64z M1.869,7.64h6.747v0.72H1.869l2.214,2.214l-0.509,0.509L0.491,8l3.083-3.083l0.509,0.509L1.869,7.64z"
  })), children);
});

module.exports = Firewall;
