// @carbon/pictograms-react/lib/reliability/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var Icon = require('../Icon-9b18a60b.js');
var React = require('react');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;

var _excluded = ["children"];
var Reliability = /*#__PURE__*/React__default["default"].forwardRef(function Reliability(_ref, ref) {
  var children = _ref.children,
      rest = Icon._objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React__default["default"].createElement(Icon.Icon, Icon._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,30.36C8.082,30.36,1.64,23.918,1.64,16\tS8.082,1.64,16,1.64S30.36,8.082,30.36,16S23.918,30.36,16,30.36z M16,2.36C8.479,2.36,2.36,8.479,2.36,16S8.479,29.64,16,29.64\tS29.64,23.522,29.64,16C29.64,8.479,23.521,2.36,16,2.36z M14.014,22.522l-4.268-4.268l0.509-0.51l3.732,3.732l8.746-9.718\tl0.535,0.481L14.014,22.522z"
  })), children);
});

module.exports = Reliability;
