// @carbon/pictograms-react/lib/secure--hybrid--cloud/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var Icon = require('../Icon-9b18a60b.js');
var React = require('react');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;

var _excluded = ["children"];
var SecureHybridCloud = /*#__PURE__*/React__default["default"].forwardRef(function SecureHybridCloud(_ref, ref) {
  var children = _ref.children,
      rest = Icon._objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React__default["default"].createElement(Icon.Icon, Icon._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M19,31.36c-3.507,0-6.36-2.854-6.36-6.36v-7.223l6.36-3.18l6.36,3.18V25\tC25.36,28.507,22.507,31.36,19,31.36z M13.36,18.223V25c0,3.11,2.53,5.64,5.64,5.64s5.64-2.529,5.64-5.64v-6.777L19,15.402\tL13.36,18.223z M27.112,20.342c2.382-0.779,4.248-3.604,4.248-6.429c0-3.135-2.366-5.774-5.443-6.143\tc-0.367-4.023-3.739-7.13-7.81-7.13c-2.918,0-5.614,1.649-6.959,4.23c-0.857-0.31-1.753-0.468-2.667-0.468\tc-4.324,0-7.841,3.523-7.841,7.853c0,4.104,2.845,7.587,6.667,8.105H11c0.199,0,0.36-0.161,0.36-0.36s-0.161-0.36-0.36-0.36\tl-3.644,0.004C3.994,19.186,1.36,15.94,1.36,12.255c0-3.933,3.195-7.133,7.122-7.133c0.932,0,1.842,0.18,2.707,0.535\tc0.175,0.073,0.381-0.006,0.463-0.181c1.17-2.5,3.705-4.117,6.457-4.117c3.791,0,6.916,2.969,7.113,6.758\tc0.01,0.182,0.153,0.328,0.335,0.341c2.851,0.195,5.084,2.59,5.084,5.454c0,2.533-1.647,5.056-3.752,5.746\tc-0.188,0.062-0.292,0.265-0.229,0.454c0.05,0.151,0.19,0.248,0.342,0.248C27.037,20.36,27.075,20.354,27.112,20.342z"
  })), children);
});

module.exports = SecureHybridCloud;
