// @carbon/pictograms-react/es/threat--management/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _objectWithoutProperties, I as Icon, a as _extends } from '../Icon-ca73eb7b.js';
import React from 'react';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;

var _excluded = ["children"];
var ThreatManagement = /*#__PURE__*/React.forwardRef(function ThreatManagement(_ref, ref) {
  var children = _ref.children,
      rest = _objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16,31.36C9.736,31.36,4.64,26.264,4.64,20V6.786L16,0.59l11.36,6.196V20\tC27.36,26.264,22.264,31.36,16,31.36z M5.36,7.214V20c0,5.867,4.773,10.64,10.64,10.64S26.64,25.868,26.64,20V7.214L16,1.41\tL5.36,7.214z M16.36,24h-0.72v-6h0.72V24z M24,16.36h-6v-0.72h6V16.36z M14,16.36H8v-0.72h6V16.36z M16.36,14h-0.72V8h0.72V14z M14,22.1c-0.041,0-0.082-0.007-0.122-0.021c-1.824-0.653-3.303-2.133-3.956-3.957c-0.067-0.187,0.03-0.393,0.217-0.46\tc0.189-0.068,0.393,0.03,0.46,0.218c0.581,1.623,1.897,2.939,3.521,3.521c0.187,0.067,0.284,0.273,0.217,0.46\tC14.286,22.008,14.147,22.1,14,22.1z M18,22.1c-0.147,0-0.286-0.092-0.339-0.239c-0.067-0.187,0.03-0.393,0.218-0.46\tc1.623-0.582,2.939-1.898,3.521-3.521c0.066-0.188,0.273-0.287,0.46-0.218c0.188,0.067,0.285,0.273,0.218,0.46\tc-0.653,1.824-2.133,3.304-3.957,3.957C18.081,22.093,18.04,22.1,18,22.1z M10.261,14.36c-0.041,0-0.081-0.006-0.122-0.021\tc-0.187-0.067-0.284-0.273-0.217-0.46c0.653-1.824,2.132-3.303,3.956-3.956c0.188-0.068,0.394,0.03,0.46,0.217\ts-0.03,0.394-0.217,0.46c-1.624,0.582-2.94,1.897-3.521,3.521C10.547,14.269,10.409,14.36,10.261,14.36z M21.739,14.36\tc-0.147,0-0.286-0.091-0.339-0.238c-0.582-1.624-1.898-2.94-3.521-3.521c-0.188-0.067-0.285-0.273-0.218-0.46\ts0.271-0.285,0.46-0.217c1.824,0.653,3.304,2.132,3.957,3.956c0.067,0.188-0.03,0.394-0.218,0.46\tC21.82,14.354,21.779,14.36,21.739,14.36z"
  })), children);
});

export { ThreatManagement as default };
