// @carbon/pictograms-react/lib/mobile--phone/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var Icon = require('../Icon-9b18a60b.js');
var React = require('react');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;

var _excluded = ["children"];
var MobilePhone = /*#__PURE__*/React__default["default"].forwardRef(function MobilePhone(_ref, ref) {
  var children = _ref.children,
      rest = Icon._objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React__default["default"].createElement(Icon.Icon, Icon._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M25,31.36H7c-0.199,0-0.36-0.161-0.36-0.36V1c0-0.199,0.161-0.36,0.36-0.36h18\tc0.199,0,0.36,0.161,0.36,0.36v30C25.36,31.199,25.199,31.36,25,31.36z M7.36,30.64h17.28v-6.28H7.36V30.64z M7.36,23.64h17.28V1.36\tH7.36V23.64z M17,27.5c0-0.552-0.448-1-1-1s-1,0.448-1,1s0.448,1,1,1S17,28.052,17,27.5z"
  })), children);
});

module.exports = MobilePhone;
