// @carbon/pictograms-react/es/this-side-up/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _objectWithoutProperties, I as Icon, a as _extends } from '../Icon-ca73eb7b.js';
import React from 'react';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;

var _excluded = ["children"];
var ThisSideUp = /*#__PURE__*/React.forwardRef(function ThisSideUp(_ref, ref) {
  var children = _ref.children,
      rest = _objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36H1v-0.72h30V31.36z M13.254,6.746L7,0.491L0.746,6.746l0.509,0.509L6.64,1.869V26h0.72V1.869\tl5.386,5.385L13.254,6.746z M31.255,6.746L25,0.491l-6.255,6.255l0.51,0.509L24.64,1.87V26h0.721V1.87l5.385,5.385L31.255,6.746z"
  })), children);
});

export { ThisSideUp as default };
