// @carbon/pictograms-react/lib/speedometer/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var Icon = require('../Icon-9b18a60b.js');
var React = require('react');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;

var _excluded = ["children"];
var Speedometer = /*#__PURE__*/React__default["default"].forwardRef(function Speedometer(_ref, ref) {
  var children = _ref.children,
      rest = Icon._objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React__default["default"].createElement(Icon.Icon, Icon._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M28.811,26.812l-4.33-2.5l0.359-0.623l4.33,2.5L28.811,26.812z M3.19,26.812l-0.36-0.623l4.33-2.5l0.36,0.623L3.19,26.812z M31,19.36h-5v-0.72h5V19.36z M6,19.36H1v-0.72h5V19.36z M16.18,19.312l-0.359-0.623l13-7.5l0.359,0.624L16.18,19.312z M7.16,14.312l-4.33-2.5l0.36-0.624l4.33,2.5L7.16,14.312z M21.312,10.52l-0.623-0.36l2.5-4.33l0.623,0.36L21.312,10.52z M10.688,10.52l-2.5-4.33l0.624-0.36l2.5,4.33L10.688,10.52z M16.36,9h-0.72V4h0.72V9z"
  })), children);
});

module.exports = Speedometer;
