// @carbon/pictograms-react/lib/coat--hanger/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var Icon = require('../Icon-9b18a60b.js');
var React = require('react');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;

var _excluded = ["children"];
var CoatHanger = /*#__PURE__*/React__default["default"].forwardRef(function CoatHanger(_ref, ref) {
  var children = _ref.children,
      rest = Icon._objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React__default["default"].createElement(Icon.Icon, Icon._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M29.38,23.114l-13.02-9.3V11.34c1.684-0.18,3-1.61,3-3.341c0-1.853-1.508-3.36-3.36-3.36\tS12.64,6.147,12.64,8h0.72c0-1.456,1.185-2.64,2.64-2.64c1.456,0,2.64,1.185,2.64,2.64s-1.184,2.64-2.64,2.64\tc-0.199,0-0.36,0.161-0.36,0.36v2.814l-13.02,9.3c-0.614,0.438-0.98,1.151-0.98,1.905c0,1.291,1.05,2.341,2.34,2.341h24.04\tc1.291,0,2.341-1.05,2.341-2.341C30.36,24.265,29.994,23.553,29.38,23.114z M28.02,26.64H3.98c-0.894,0-1.621-0.727-1.621-1.62\tc0-0.522,0.254-1.016,0.679-1.319L16,14.442L28.962,23.7c0.425,0.304,0.678,0.797,0.678,1.319\tC29.64,25.913,28.913,26.64,28.02,26.64z"
  })), children);
});

module.exports = CoatHanger;
