// @carbon/pictograms-react/lib/encryption/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var Icon = require('../Icon-9b18a60b.js');
var React = require('react');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;

var _excluded = ["children"];
var Encryption = /*#__PURE__*/React__default["default"].forwardRef(function Encryption(_ref, ref) {
  var children = _ref.children,
      rest = Icon._objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React__default["default"].createElement(Icon.Icon, Icon._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    fill: "none",
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeMiterlimit: "10",
    strokeWidth: ".72",
    d: "M10,21c0-3.314,2.686-6,6-6s6,2.686,6,6s-2.686,6-6,6S10,24.314,10,21z M16,11c-5.523,0-10,4.477-10,10s4.477,10,10,10\ts10-4.477,10-10S21.523,11,16,11z M11,17.682V6c0-2.761,2.239-5,5-5s5,2.239,5,5v11.682 M16,22v2 M16,20c-0.552,0-1,0.448-1,1\ts0.448,1,1,1s1-0.448,1-1S16.552,20,16,20z"
  })), children);
});

module.exports = Encryption;
