// @carbon/pictograms-react/lib/home--garage/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var Icon = require('../Icon-9b18a60b.js');
var React = require('react');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;

var _excluded = ["children"];
var HomeGarage = /*#__PURE__*/React__default["default"].forwardRef(function HomeGarage(_ref, ref) {
  var children = _ref.children,
      rest = Icon._objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React__default["default"].createElement(Icon.Icon, Icon._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H1v-0.72h1.64V19h0.72v11.64h4.28V19c0-0.199,0.161-0.36,0.36-0.36h16\tc0.199,0,0.36,0.161,0.36,0.36v11.64h4.279V19h0.721v11.64H31V31.36z M8.36,30.64h15.28v-3.28L8.36,27.363V30.64z M8.36,23.363\tv3.279l15.28-0.002v-3.28L8.36,23.363z M8.36,19.36v3.282l15.28-0.002v-3.28H8.36z M8,16.363v-0.72l16-0.003v0.72L8,16.363z M30.745,16.254L16,1.509L1.254,16.254l-0.509-0.509l15-15c0.141-0.141,0.368-0.141,0.509,0L28.64,13.131V9h0.721v4.851l1.895,1.895\tL30.745,16.254z"
  })), children);
});

module.exports = HomeGarage;
