// @carbon/pictograms-react/lib/text--input/index.js
/**
 * Copyright IBM Corp. 2019, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var Icon = require('../Icon-9b18a60b.js');
var React = require('react');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;

var _excluded = ["children"];
var TextInput = /*#__PURE__*/React__default["default"].forwardRef(function TextInput(_ref, ref) {
  var children = _ref.children,
      rest = Icon._objectWithoutProperties(_ref, _excluded);

  return /*#__PURE__*/React__default["default"].createElement(Icon.Icon, Icon._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,29.36H2c-0.199,0-0.36-0.161-0.36-0.36v-4c0-0.199,0.161-0.36,0.36-0.36h28\tc0.199,0,0.36,0.161,0.36,0.36v4C30.36,29.199,30.199,29.36,30,29.36z M2.36,28.64h27.28v-3.28H2.36V28.64z M14,23.36H2v-0.72h12\tV23.36z M30,19.36H2c-0.199,0-0.36-0.161-0.36-0.36v-4c0-0.199,0.161-0.36,0.36-0.36h28c0.199,0,0.36,0.161,0.36,0.36v4\tC30.36,19.199,30.199,19.36,30,19.36z M2.36,18.64h27.28v-3.28H2.36V18.64z M14,13.36H2v-0.72h12V13.36z M30,9.36H2\tC1.801,9.36,1.64,9.199,1.64,9V5c0-0.199,0.161-0.36,0.36-0.36h28c0.199,0,0.36,0.161,0.36,0.36v4C30.36,9.199,30.199,9.36,30,9.36z M2.36,8.64h27.28V5.36H2.36V8.64z M14,3.36H2V2.64h12V3.36z"
  })), children);
});

module.exports = TextInput;
